src/script/cpp_api/s_security.cpp

```diff
@@ -623,21 +666,6 @@ bool ScriptApiSecurity::checkPath(lua_State *L, const char *path,
 	return false;
 }
```

Comment:
thought: this function should be very reasonably unit-testable
your choice if you want to work on that.

Reply:
This is kind of a pain as the function accesses `L` and a bunch of stuff stored in there, like `IGameDef` and the mod name. Some code also requires accessing the filesystem, so would need to depend on devtest.

Here's a quick implementation: https://github.com/minetest/minetest/commit/a75a33246286a4dcb0f1ddbfc17c8ecb4a6a54ab

Reply:
I was thinking of this less narrowly.
A `Scripting` instance and mock `IGameDef` is easily created and setting up a temporary folder for the right test situations should be reasonably easy.
But it's definitely okay to leave this for later.